cvmfs/nfs_shared_maps.cc

```diff
@@ -206,20 +208,17 @@ bool GetPath(const uint64_t inode, PathString *path) {
   path->Assign(raw_path, strlen(raw_path));
   sqlite3_reset(stmt_get_path_);
   pthread_mutex_unlock(&lock_);
-  atomic_inc64(&dbstat_inode_found_);
+  perf::Inc(n_dbstat_inode_found_);
   return true;
 }
 
 
 string GetStatistics() {
```

Comment:
It is used in swissknife_pull.cc, line 558. However, it's just a log that can be perfectly removed